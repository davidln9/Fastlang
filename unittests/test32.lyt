
int main() {

	g_val1 = 3;
	g_val2 = 5;

	println("g_val1: " + g_val1);
	println("g_val2: " + g_val2);

	doSwap();

	println("g_val1: " + g_val1);
	println("g_val2: " + g_val2);

}


int doSwap() {

	int tmp = g_val1;
	g_val1 = g_val2;
	g_val2 = tmp;
	return 0;
}

int g_val1;
int g_val2;
