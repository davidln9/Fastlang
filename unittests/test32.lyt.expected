g_val1: 3
g_val2: 5
g_val1: 5
g_val2: 3
